packages/launcher/test/unit/linux_spec.ts

```diff
@@ -42,6 +43,24 @@ describe('linux browser detection', () => {
     return linuxHelper.detect(goal).then(checkBrowser)
   })
 
+  // https://github.com/cypress-io/cypress/issues/6669
+  it('detects browser if the --version stdout is multiline', () => {
+    stdout.withArgs('multiline-foo', ['--version'])
+    .resolves('Running without a11y support!\nfoo-browser v9001.1.2.3')
```

Comment:
can you please just use string template (strip indent) instead of `\n`, since that one is so simple to miss.

Reply:
eh... every time i use the common-tags, i have to manually check that they do what i expect, since i've been burned before by assuming that they do what it says on the can - see "stripIndent" and "stripIndents" doing different things, for example

Reply:
it also helps that vscode highlights escape sequences for you, so in an editor, this will be clear. looks like github's syntax highlighting isn't there yet :/